lib/matplotlib/backends/backend_qt5.py

```diff
@@ -379,13 +379,15 @@ def keyReleaseEvent(self, event):
             FigureCanvasBase.key_release_event(self, key, guiEvent=event)
 
     @property
+    @cbook.deprecated("3.0", "Manually check `event.isAutoRepeat()`")
```

Comment:
This is a method on the guiEvent property that makes it through, not on the `backend_bases.Event` instance that makes it to the user.